pytket/docs/conf.py

```diff
@@ -60,6 +60,7 @@
     "sphinx_copybutton",
     "jupyter_sphinx",
```

Comment:
The jupyter sphinx cells have quite sharp edges in the new theme. See the docs for `pytket.circuit.display`

![Screenshot 2024-06-27 at 17 07 05](https://github.com/CQCL/tket/assets/93673602/14e87d00-4a62-47f2-8a1d-6309888f11c8)

Compare with 

https://tket.quantinuum.com/api-docs/display.html#example-usage


We do use these cells a lot in the manual so it'd be nice to improve the look of them somehow.

Reply:
can discuss with other teams